src/precice/impl/CommonErrorMessages.hpp

```diff
@@ -16,6 +16,8 @@ inline std::string errorInvalidEdgeID(int eid)
   return "The given EdgeID \"" + std::to_string(eid) + "\" is invalid. Check that it originated from a call to setMeshEdge().";
 }
 
+static constexpr auto errorInvalidVertexIDRange = "The given range of VertexIDs contains invalid IDs at offsets [{},{}). Check that they originated from calls to setMeshVertex() or setMeshVertices().";
```

Comment:
```suggestion
static constexpr auto errorInvalidVertexIDRange = "The given range of VertexIDs contains invalid IDs at offsets [{},{}]. Check that they originated from calls to setMeshVertex() or setMeshVertices().";
```
... could be easier to understand

Reply:
It's an open interval though :wink: so either `[A,B[` or `[A,B)`. The cppreference even uses the `)` version, so I consider it fairly wide-spread.

Reply:
> We could of course use a closed interval instead.

That's what I had in mind. As user I would find a closed interval easier to understand.